src/content/en/updates/2017/12/chrome-loadtimes-deprecated.md

```diff
@@ -0,0 +1,358 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The non-standard chrome.loadTimes() API will be deprecated in Chrome 64 now that standards-based equivalents exist for all of its features.
+
+{# wf_updated_on: 2017-12-14 #}
+{# wf_published_on: 2017-12-14 #}
+{# wf_tags: performance #}
+{# wf_blink_components: Blink>PerformanceAPIs #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: The non-standard chrome.loadTimes() API will be deprecated in Chrome 64 now that standards-based equivalents exist for all of its features. #}
+{# wf_blink_components: Blink>PerformanceAPIs #}
+
+# Chrome 64 to deprecate the chrome.loadTimes() API {: .page-title }
+
+{% include "web/_shared/contributors/philipwalton.html" %}
+
+`chrome.loadTimes()` is a non-standard API that exposes loading metrics and
+network information to developers in order to help them better understand their
+site's performance in the real world.
+
+Since this API was implemented in 2009, all of the information it reports can be
+found in standardized APIs such as [Navigation Timing 2](https://www.w3.org/TR
+/navigation-timing-2/), [Paint Timing](https://www.w3.org/TR/paint-timing/), and
+the recent [`nextHopProtocol`](https://www.w3.org/TR/resource-timing-2/#dom-
+performanceresourcetiming-nexthopprotocol) addition to Navigation Timing 2 and
+[Resource Timing 2](https://www.w3.org/TR/resource-timing-2/).
+
+These standardized APIs are being implemented by multiple browser vendors. As a
+result, `chrome.loadTimes()` is being deprecated in Chrome 64.
+
+## The deprecated API
+
+The `chrome.loadTimes()` function returns a single object containing all of its
+loading and network information. For example, the following object is the result
+of calling `chrome.loadTimes()` on [www.google.com](https://www.google.com):
+
+```
+{
+  "requestTime": 1513186741.847,
+  "startLoadTime": 1513186741.847,
+  "commitLoadTime": 1513186742.637,
+  "finishDocumentLoadTime": 1513186742.842,
+  "finishLoadTime": 1513186743.582,
+  "firstPaintTime": 1513186742.829,
+  "firstPaintAfterLoadTime": 0,
+  "navigationType": "Reload",
+  "wasFetchedViaSpdy": true,
+  "wasNpnNegotiated": true,
+  "npnNegotiatedProtocol": "h2",
+  "wasAlternateProtocolAvailable": false,
+  "connectionInfo": "h2"
+}
+```
+
+## Standardized replacements
+
+Each of the above values can now be found using standardized APIs. The following
```

Comment:
'Each of the above values can now be found' -> 'You can now find each of the above values'.

(Changing from passive to active voice and directly addressing the reader both improve comprehension.)

Reply:
Done.